### ISSUE TITLE
Hebrew keyboard support

### ISSUE THREAD

**Issue by jloewenstein** [EzerIT/BibleOL#48]
Shalom,
For me, and my students it would be beneficial to be able to type in Hebrew using the standard Hebrew keyboard. 
Currently trying to type into a field put in values according to the phonetic Hebrew keyboard.
Is there a way to display and type using the [standard Hebrew Keyboard](https://he.wikipedia.org/wiki/%D7%9E%D7%A7%D7%9C%D7%93%D7%AA_%D7%A2%D7%91%D7%A8%D7%99%D7%AA)?

**Comment by tmccormack165** [comment]
Hello @jloewenstein, thank you for posting this, I do believe this is possible, and I will notify you when this is completed.

**Comment by ernstboogert** [comment]
We have had a virtual keyboard in the past. However, students found it very difficult to use. It was also almost impossible to use on mobile phones. I think the code is still there and maybe we can offer it as a secondary option. Not sure though whether it really helps... We will think about it. Thanks for bringing up the question!

**Comment by ernstboogert** [comment]
> Hello @jloewenstein, thank you for posting this, I do believe this is possible, and I will notify you when this is completed.

Lets discuss it in the team first... It is part of a bigger discussion.

**Comment by ernstboogert** [comment]
> Shalom,
> For me, and my students it would be beneficial to be able to type in Hebrew using the standard Hebrew keyboard. 
> Currently trying to type into a field put in values according to the phonetic Hebrew keyboard.
> Is there a way to display and type using the [standard Hebrew Keyboard](https://he.wikipedia.org/wiki/%D7%9E%D7%A7%D7%9C%D7%93%D7%AA_%D7%A2%D7%91%D7%A8%D7%99%D7%AA)?
> 

I think you can already put your keyboard settings in Hebrew and type the Hebrew letters in an input field. Or use a Hebrew keyboard...

**Comment by jloewenstein** [comment]
Yes, it does function, but not with the standard Hebrew keyboard. 
what pops up is a phonology based keyboard. 
![image](https://github.com/EzerIT/BibleOL/assets/163320105/89f68bab-19f1-4e4e-b698-2e0a440b27b8)
In this case I typed on my keyboard: D S A which in a standard Hebrew keyboard should output ג(D) ד(S) ש(A) (not דסַ)

**Comment by ernstboogert** [comment]
@jloewenstein Ah, now I understand your point. I think we have to look into adding other keyboards to VirtualKeyboard... There might be other options that can easily be added...

**Comment by jloewenstein** [comment]
Is it ok to resurface the question?
